rose/web/game.js

```diff
@@ -1,7 +1,6 @@
-var ROSE = (function() {
-    "use strict";
 
-    function App() {
+class App {
+    constructor() {
```

Comment:
You might prefer to use class fields instead of constructor assignments - it helps with certain runtime optimizations and with static analysis. There are subtle runtime differences (set vs define semantics) but in this case you probably won't encounter them

Reply:
changed to class fields